integration-testing/election-manager/cypress/integration/configuration.ts

```diff
@@ -0,0 +1,20 @@
+describe('Election Manager and Module Converter MS SEMS configuration', () => {
+  it('Election Manager can be configured with MS SEMS files', () => {
+    cy.visit('/')
+    cy.contains('Convert from SEMS files').click()
+    cy.get('input[name="SEMS main file"]').attachFile('semsMainFile.txt')
+    cy.get('input[name="SEMS candidate mapping file"]').attachFile(
+      'semsCandidateMappingFile.txt'
+    )
+    cy.contains('Election loading')
+    cy.contains('Special Election for Senate 15')
+    cy.contains('Election Hash: f30480cc99')
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(500)
```

Comment:
Why?

Reply:
I'll add a comment, the page re-renders twice quickly when it first loads (I haven't dug extensively into why) and I was running into a test flakiness issue where the test would grab the "Definition" button before the rerender happens so when it tries to click it has been detached from the dom. I think that is what is happening in this CI failure: https://app.circleci.com/pipelines/github/votingworks/vxsuite/1555/workflows/2987319e-fcfc-45a5-9a0a-9477d0d66a14/jobs/17622